app/utils/store.py

```diff
@@ -63,5 +63,6 @@ def get(self, service_id, document_id, decryption_key):
     def generate_encryption_key(self):
         return os.urandom(32)
 
-    def get_document_key(self, service_id, document_id):
-        return "{}/{}".format(service_id, document_id)
+    def get_document_key(self, service_id, document_id, sending_method=None):
+        key_prefix = 'tmp/' if sending_method == 'attach' else ''
+        return "{}{}/{}".format(key_prefix, service_id, document_id)
```

Comment:
```suggestion
        return f"{key_prefix}{service_id}/{document_id}".
```
My quest
![image](https://user-images.githubusercontent.com/295709/112362292-aa095780-8caa-11eb-8025-d45cad6667e9.png)